app/assets/stylesheets/_accessible-autocomplete.scss

```diff
@@ -1,16 +1,16 @@
 .autocomplete__wrapper,
 .autocomplete__input,
 .autocomplete__hint {
-  font-family: $govuk-font-family;
+  font-family: govuk-font-family;
```

Comment:
Why this change? This Sass variable should update the font stack to the one used by the design system (`"GDS Transport", arial, sans-serif;`). I’d be wary of changing it.

Reply:
You're right this was a mistake, should be fixed now though.